pkg/drivers/generic/generic.go

```diff
@@ -49,7 +49,7 @@ var (
 				SELECT MAX(mkv.id) AS id
 				FROM kine AS mkv
 				WHERE
-					mkv.name LIKE ?
+					mkv.name LIKE ? ESCAPE '\'
```

Comment:
@brandond

```
time="2025-12-09T01:22:09.73174818Z" level=fatal msg="starting kine backend: Error 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''\\' AND\n\t\t\t\tkv.id > ?\n\t\t\tORDER BY kv.id ASC' at line 9"
```

The ESCAPE keyword may not be supported in MySQL, I tested work without `ESCAPE '\'`, `\` seems to be the default MySQL escape symbol.

Reply:
Hmm, that shouldn't have passed tests if so...

Reply:
Ah I see, we lost Drone so we currently have almost no tests. I'll have to fix that as well.